### ISSUE TITLE
Bencher up doesn't work

### ISSUE THREAD

**Issue by ShtiviOmer** [bencherdev/bencher#541]
Hello,
We are trying to use bencher self-hosted mode, we followed the tutorial on how to start.

Bencher cli is installed:
``` bash
 bencher --version
bencher 0.4.29
```
  
And docker is installed:
``` bash
docker --version
Docker version 27.3.1, build ce1223035a
```

When running:
``` bash
bencher up
Pulling `ghcr.io/bencherdev/bencher-console:v0.4.29` image...
Pulling `ghcr.io/bencherdev/bencher-api:v0.4.29` image...
Creating `bencher_api` container...
Starting `bencher_api` container...
.. omitted
node:internal/modules/cjs/loader:1228

  throw err;

  ^



Error: Cannot find module '/usr/src/bencher/dist/server/entry.mjs'

    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)

    at Module._load (node:internal/modules/cjs/loader:1051:27)

    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)

    at node:internal/main/run_main_module:28:49 {

  code: 'MODULE_NOT_FOUND',

  requireStack: []

}



Node.js v20.18.0
```

When trying to access:
``` bash
 curl http://127.0.0.1:3000
curl: (7) Failed to connect to 127.0.0.1 port 3000 after 0 ms: Couldn't connect to server
```

**Comment by epompeii** [comment]
@ShtiviOmer sorry you're running into trouble.
I'll try to take a look soon and see if I can replicate what you're seeing.

**Comment by ShtiviOmer** [comment]
Hi Everett
Thank you for your response
I’m using apple silicon ( m3 ) so maybe the bin isn’t compiled for this cpu
?

**Comment by epompeii** [comment]
Hey @ShtiviOmer, I've been able to repro the issue on an M1.
This is definitely a bug on my end. This patch should not have been commited: https://github.com/bencherdev/bencher/commit/0a6b5a4644226a9ee1591a4be5fb9f513b55bbe0#diff-b94485bcf4068ea26eeebe42c740ab6c00b4f95ecc99d070f3608578813191c6
It is now reverted here: https://github.com/bencherdev/bencher/commit/3a2eb247493b65fd2489456c435d85a37888d67f

The new release should be out shortly.
